polyhost/gui/log_bundle_dialog.py

```
@@ -0,0 +1,222 @@
+            return True, f"Copied {lines} lines to the clipboard.", text
+
+        self._start(work)
+
+    def _reveal(self):
+        """Show the saved bundle in the OS file manager."""
+        path = self._last_bundle
+        if not path or not os.path.exists(path):
+            return
+        try:
+            if sys.platform.startswith("darwin"):
+                subprocess.run(["open", "-R", path], check=False)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)